### DISCUSSION TITLE
Access external hard drives of the server where code-server is installed?

### DISCUSSION THREAD

**Discussion by nipunarora** [coder/code-server#4823] [Q&A] [answered]
I have code server installed on my mac mini, and I'd like to develop on projects hosted in this mac mini via the web. However, I can't seem to access any files in the mac mini filesystem. What do I have to add to the configuration to enable this?

**Comment by nipunarora** [comment]
Actually I should be more specific here, what it's really not being able to access is external hard drives connected to the system. I can see the mount point in the file list when i try to open the folder, but there isn't anything in it.

**Comment by nipunarora** [comment, marked as answer]
This question seemed like duplicate of - https://github.com/coder/code-server/discussions/4206 
Works :) !

**Reply by jsjoeio** [reply]
Hooray!
